docker/openproblems-r-base/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get -yq --no-install-suggests --no-install-recommends install dirmngr ca
 RUN apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
 RUN echo 'deb http://cloud.r-project.org/bin/linux/debian stretch-cran35/' >> /etc/apt/sources.list
 RUN apt-get update -qq
-RUN apt-get install -yq --no-install-suggests --no-install-recommends r-base-core=3.6\*
+RUN apt-get install -yq --no-install-suggests --no-install-recommends r-base-core=3.6\* r-base-dev
```

Comment:
Should we be specifying a version for r-base-dev? And if r-base-core is a dependency for r-base-dev, can we drop the former?

Reply:
:+1:

In the future, we'll probably have to switch to R 4.0 because some maintainers of R packages tend to drop support for older R versions almost immediately. If this becomes an issue, we can always copy paste some sections of my randpy dockerfiles, e.g. https://github.com/data-intuitive/randpy/blob/master/dockerfiles/r4.0_py3.7.Dockerfile.